pkg/natdiscovery/proto/natdiscovery.proto

```diff
@@ -18,16 +18,16 @@ limitations under the License.
 syntax = "proto3";
 option go_package = "github.com/submariner-io/submariner/pkg/natdiscovery/proto";
 
-message SubmarinerNatDiscoveryMessage {
+message SubmarinerNATDiscoveryMessage {
```

Comment:
Oh, did it regenerate the natdiscovery.pb.go fine with those changes?

Reply:
it did :)

Reply:
yes. I tried it locally. It changes the license, import sequence and some bytes of `file_pkg_natdiscovery_proto_natdiscovery_proto_rawDesc` var though.

I modified import sequence and license to match as it was before manually